api/index.d.ts

```diff
@@ -3609,7 +3609,7 @@ declare module "socket:dgram" {
      */
     export class Socket extends EventEmitter {
         constructor(options: any, callback: any);
-        id: bigint;
+        id: any;
```

Comment:
hmm is this loose typing because of `this.id = options?.id || rand64()`?
can we type options bag?